Changes.md

```diff
@@ -18,6 +18,7 @@ Fixes
 -----
 
 - ScriptNode : Fixed bugs that allowed global variables to remain in the context after they had been disabled, renamed or deleted.
+- OSL : Fixed warnings about using deprecated form of get_context
```

Comment:
We don't need to include this in the release notes, since the warning has never been user-visible, at least not in standard gaffer builds.